src/FlightDisplay/TelemetryValuesBar.qml

```diff
@@ -20,7 +20,7 @@ Rectangle {
     id:                 telemetryPanel
     height:             telemetryLayout.height + (_toolsMargin * 2)
     width:              telemetryLayout.width + (_toolsMargin * 2)
-    color:              Qt.hsla(_baseBGColor.hslHue, _baseBGColor.hslSaturation, _baseBGColor.hslLightness, 0.5)
+    color:              Qt.hsla(_baseBGColor.hslHue, _baseBGColor.hslSaturation, _baseBGColor.hslLightness, 1)
     radius:             ScreenTools.defaultFontPixelWidth / 2
 
     property color _baseBGColor: qgcPal.window
```

Comment:
Which also means this is no longer needed

Reply:
Fixed and rebased to master!